Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -550,6 +550,11 @@ void MenuBar::AddHelpMenu()
   connect(github, &QAction::triggered, this, []() {
     QDesktopServices::openUrl(QUrl(QStringLiteral("https://github.com/dolphin-emu/dolphin")));
   });
+  QAction* bugtracker = help_menu->addAction(tr("&Bug Tracker"));
+  connect(bugtracker, &QAction::triggered, this, []() {
```

Comment:
Pretty sure `this` is unneeded here, since you're only executing this lambda.

Reply:
I just copied the layout of the other buttons, I think it's cleaner to keep them all the same.

Reply:
I don't necessarily agree, but it can also be corrected in bulk in the future. No big deal.